### DISCUSSION TITLE
Release 1.1

### DISCUSSION THREAD

**Discussion by sumeshir26** [sumeshir26/TimerX#104] [Announcements]
Release 1.1 (Add your content here and then publish this.)

| ⚠️ Warning: You will need to uninstall the previous version on TimerX if you have it. You won't face this problem from the next release
| -

## ✨What's New
- More settings by @not-nef
- Linux Support by @rdbende
- Accurate timer by @sumeshir26
- Notification when an update is available by @sumeshir26 and @im-coder-lg 
- Mica effect by @sumeshir26 
- Plenty of bug fixes and code cleanup!

## New Contributors
* @rdbende made their first contribution in https://github.com/Futura-Py/TimerX/pull/88

### 🌴Enjoy!

**Full Changelog**: https://github.com/Futura-Py/TimerX/compare/1.0...1.1

<hr /><em>This discussion was created from the release <a href='https://github.com/Futura-Py/TimerX/releases/tag/1.1'>Release 1.1</a>.</em>

**Comment by not-nef** [comment]
@sumeshir26 Why doesn't it contain an msi?

**Comment by sumeshir26** [comment]
It does- the 17 mb one

**Reply by not-nef** [reply]
![image](https://user-images.githubusercontent.com/83908932/157863437-7d240579-5820-4c0a-9f9c-5d8f4d0a22a6.png)

**Reply by lg-nineoneninethreenine** [reply]
Just popping in during exam revision - @sumeshir26 are you referring to the old draft releases? Also, you could have made a team discussion, rather than a release one, right?

**Comment by lg-nineoneninethreenine** [comment]
You didn't use the workflow! Use that, it uploads the assets as needed.
^_^

**Comment by rdbende** [comment]
Every asset is included, probably only the mobile app does not show it
![image](https://user-images.githubusercontent.com/77941087/157865036-ffdf31c7-77c2-4f5a-968a-7b08d9f6e1a2.png)

**Reply by lg-nineoneninethreenine** [reply]
It shows the 1.0 assets. Nothing is wrong on my side. Maybe it's a new update :thinking:

**Comment by sumeshir26** [comment]
No, i tested the msi, it works. The linux binaries probably dont work since we never testede them
